examples/fabric/reinforcement_learning/rl/utils.py

```diff
@@ -119,7 +117,12 @@ def parse_args():
     return args
 
 
-def layer_init(layer: torch.nn.Module, std: float = np.sqrt(2), bias_const: float = 0.0, ortho_init: bool = True):
+def layer_init(
+    layer: torch.nn.Module,
+    std: float = torch.sqrt(torch.tensor(2)).item(),
```

Comment:
how about `math.sqrt(2)`?

Reply:
Changed, thanks. (I was being silly and didn't want to introduce the math import when I was trying to remove the numpy import)

Reply:
Since math is a built-in module of Python, this is not a problem at all :)